sdk/search/azure-search-documents/azure/search/documents/indexes/_internal/_search_index_client.py

```diff
@@ -235,7 +235,7 @@ def create_or_update_index(
 
     @distributed_trace
     def analyze_text(self, index_name, analyze_request, **kwargs):
-        # type: (str, AnalyzeRequest, **Any) -> AnalyzeResult
+        # type: (str, AnalyzeTextRequest, **Any) -> AnalyzeResult
```

Comment:
Should be `AnalyzeTextOptions` now, or was that going to come in a separate PR? See #11830.

Reply:
Already fixed. :)